demisto_sdk/commands/init/initiator.py

```diff
@@ -118,18 +179,25 @@ def convert_contribution_to_pack(self):
                 with zipped_contrib.open('metadata.json') as metadata_file:
                     click.echo(f'Pulling relevant information from {metadata_file.name}', color=LOG_COLORS.NATIVE)
                     metadata = json.loads(metadata_file.read())
-                    pack_name = metadata.get('name', 'ContributionPack')
+                    # a name passed on the cmd line should take precedence over one pulled
+                    # from contribution metadata
+                    pack_name = self.name or self.convert_to_valid_name(metadata.get('name', 'ContributionPack'))
+                    # a description passed on the cmd line should take precedence over one pulled
+                    # from contribution metadata
+                    pack_description = self.description or metadata.get('description')
+                    metadata_dict['description'] = pack_description
                     metadata_dict['name'] = pack_name
                     metadata_dict['author'] = metadata.get('author', '')
                     metadata_dict['support'] = metadata.get('support', '')
                     metadata_dict['url'] = metadata.get('supportDetails', {}).get('url', '')
                     metadata_dict['email'] = metadata.get('supportDetails', {}).get('email', '')
+                    metadata_dict['dependencies'] = metadata.get('dependencies')  # not sure about this one
```

Comment:
?

Reply:
I saw a newly downloaded contribution zip's metadata had actually filled this field (a recent change on server I'm guessing?) so I wasn't sure whether to include it but it seemed relevant and important.